Assets/Scripts/light_variable_intensity.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class light_variable_intensity : MonoBehaviour
+{
+    public Light light;
+    public int seed;
+    public float start_intensity;
+    private float curr_intensity;
+    public float change_rate;
+    public float min_intensity;
+    public float max_intensity;
+    private System.DateTime seed_init;
```

Comment:
1. fix naming
2. There have to be no public variables in code,
if u want to expose variable in inspektor use ` [SerializeField] private float minIntensityt;`
if you want to get access from another script to variable create getter `public float MinIntensityt => minIntensityt;`
3. Try to keep SerializeField and privat fields separated